benches/plonk.rs

```diff
@@ -65,34 +66,53 @@ where
     }
 }
 
+fn kzg10_benchmarks(c: &mut Criterion) {
+    constraint_system_benchmark::<
+        <Bls12_381 as PairingEngine>::Fr,
+        EdwardsParameters,
+        KZG10<Bls12_381>,
+    >("KZG10", c);
+}
+
+fn ipa_benchmarks(c: &mut Criterion) {
+    constraint_system_benchmark::<
+        <Bls12_377 as PairingEngine>::Fr,
+        ark_ed_on_bls12_377::EdwardsParameters,
+        IPA<<Bls12_377 as PairingEngine>::G1Affine, blake2::Blake2b>,
+    >("IPA", c);
+}
+
 /// Generates full benchmark suite for compiling, proving, and verifying.
-fn constraint_system_benchmark(c: &mut Criterion) {
+fn constraint_system_benchmark<F, P, HC>(name: &str, c: &mut Criterion)
+where
+    F: PrimeField,
+    P: TEModelParameters<BaseField = F>,
+    HC: HomomorphicCommitment<F>,
+{
     let label = b"ark".as_slice();
 
-    const MINIMUM_DEGREE: usize = 5;
-    const MAXIMUM_DEGREE: usize = 19;
+    const MINIMUM_DEGREE: usize = 2;
+    const MAXIMUM_DEGREE: usize = 5;
```

Comment:
Let's use the original values to `5` to `19`

Reply:
woops